src/Core/Components/Spacer/FluentSpacer.razor.cs

```diff
@@ -0,0 +1,44 @@
+﻿// ------------------------------------------------------------------------
+// MIT License - Copyright (c) Microsoft Corporation. All rights reserved.
+// ------------------------------------------------------------------------
+
+using Microsoft.AspNetCore.Components;
+
+namespace Microsoft.FluentUI.AspNetCore.Components;
+
+/// <summary>
+/// Spacer component, used to create space between elements.
+/// </summary>
+public partial class FluentSpacer : FluentComponentBase
+{
+    /// <summary>
+    /// Gets or sets the height of the spacer when the orientation is vertical.
+    /// </summary>
+    [Parameter]
+    public string? Height { get; set; }
+
+    /// <summary>
+    /// Gets or sets the size of the spacer when the orientation is horizontal.
+    /// </summary>
+    [Parameter]
+    public string? Width { get; set; }
+
+    /// <summary>
+    /// Specify the orientation of the parent container, which can be either horizontal or vertical.
+    /// </summary>
+    [Parameter]
+    public Orientation Orientation { get; set; } = Orientation.Horizontal;
+
+    /// <summary/>
+    public string? ClassValue => DefaultClassBuilder
```

Comment:
Like in other components, can you move these 2 properties before the [Parameter] properties?